app/javascript/angular/code/filtersUtils.js

```diff
@@ -1,5 +1,46 @@
 import tippy from "tippy.js";
 import Clipboard from "clipboard";
+import moment from "moment";
+
+const endOfToday = () =>
+  moment()
+    .utc()
+    .endOf("day")
+    .format("X");
+
+const oneYearAgo = () =>
+  moment()
+    .utc()
+    .startOf("day")
+    .subtract(1, "year")
+    .format("X");
+
+export const presentMoment = () =>
+  moment()
+    .utc()
+    .format("X");
+
+export const oneHourAgo = () =>
+  moment()
+    .utc()
+    .subtract(1, "hour")
+    .format("X");
+
+export const defaultTimeFrom = isStreaming => {
+  if (isStreaming) {
```

Comment:
You could also use the ternary
```
const fnc = isStreaming =>
  isStreaming ? a() : b()
```
I'd suggest bubbling up the conditional. Reek for example [reports that as a smell](https://github.com/troessner/reek/blob/master/docs/Boolean-Parameter.md)